TidepoolKit/TAPI.swift

```diff
@@ -206,6 +206,19 @@ public class TAPI {
         }
     }
 
+    // MARK: - Info
+
+    /// Get the supported version info for apps.
```

Comment:
[nit] Just "Get the info." because it may be expanded to include much more than version info.

Reply:
Admittedly it's pretty dodgy to have just an `/info` endpoint without any context.  Seems like it could eventually be a dumping ground... 🤷

Reply:
Yep, but its been around for a long while so I doubt it will change.